GitUI/CommandsDialogs/FormMergeBranch.cs

```diff
@@ -82,14 +82,15 @@ private void OkClick(object sender, EventArgs e)
             AppSettings.DontCommitMerge = noCommit.Checked;
             ScriptManager.RunEventScripts(this, ScriptEvent.BeforeMerge);
 
-            var successfullyMerged = FormProcess.ShowDialog(this, GitCommandHelpers.MergeBranchCmd(Branches.GetSelectedText(),
-                                                                                                   fastForward.Checked,
-                                                                                                   squash.Checked,
-                                                                                                   noCommit.Checked,
-                                                                                                   _NO_TRANSLATE_mergeStrategy.Text,
-                                                                                                   allowUnrelatedHistories.Checked,
-                                                                                                   addMergeMessage.Checked ? mergeMessage.Text : null,
-                                                                                                   addLogMessages.Checked ? (int)nbMessages.Value : (int?)null));
+            var successfullyMerged = FormProcess.ShowDialog(this, GitCommandHelpers.MergeBranchCmd(
+                Branches.GetSelectedText(),
+                fastForward.Checked,
+                squash.Checked,
+                noCommit.Checked,
+                _NO_TRANSLATE_mergeStrategy.Text,
+                allowUnrelatedHistories.Checked,
+                addMergeMessage.Checked ? mergeMessage.Text : null,
+                addLogMessages.Checked ? (int)nbMessages.Value : (int?)null));
```

Comment:
I like the original more, but I guess you can't have the cake and eat it too...

Reply:
My preference is:
```csharp
            var successfullyMerged = FormProcess.ShowDialog(
                this,
                GitCommandHelpers.MergeBranchCmd(
                    Branches.GetSelectedText(),
                    fastForward.Checked,
                    squash.Checked,
                    noCommit.Checked,
                    _NO_TRANSLATE_mergeStrategy.Text,
                    allowUnrelatedHistories.Checked,
                    addMergeMessage.Checked ? mergeMessage.Text : null,
                    addLogMessages.Checked ? (int)nbMessages.Value : (int?)null
                )
            );
```

The original formatting goes to far to the right - it is hard to review (needs horizontal scrolling). It is also nonresistant to refactorings. Imagine renaming GitCommandHelpers to GitCommands - the argument list gets misaligned.

Reply:
This is pretty much what the rule in this PR achieves, although the closing paren/brace end up on the last line of code. There doesn't seem to be a rule to enforce the indentation, though perhaps @sharwell knows of one.

As I mentioned elsewhere, once a decision is reached I will use R# to sort out the indentation across the whole solution. Hopefully there won't be too many lines to the diff.